news/6741.feature

```diff
@@ -0,0 +1 @@
+Add a subcommand ``debug`` to ``pip config`` to list available configuration files and the key-value pairs defined in them
```

Comment:
```suggestion
Add a subcommand ``debug`` to ``pip config`` to list available configuration sources and the key-value pairs defined in them
```

Reply:
Fixed in https://github.com/pypa/pip/pull/8096/commits/aced666de85be927e72355a4705829fa21cc1671